apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -4213,7 +4195,7 @@ block_to_endpoint_map(Block, Options) ->
                   aetx_sign:meta_data_from_client_serialized(Encoding, EncodedTx),
             {BlockHeight, TxBlockHeight} = {TxBlockHeight, BlockHeight},
             {BlockHash, TxBlockHash} = {TxBlockHash, BlockHash},
-            TxHash = aetx:hash(aetx_sign:tx(SignedTx)),
+            TxHash = aetx_sign:hash(SignedTx),
```

Comment:
why did you decided to move it out from aetx?

Reply:
[This comes from PR 1132](https://github.com/aeternity/epoch/pull/1132/files?utf8=%E2%9C%93&diff=unified#diff-14dc0eab04e011248c7640cb3c1cf0deR4198) this PR depends upon (and that is included in this PR - I updated description clarifying inclusion).